unstructured/documents/ontology.py

```diff
@@ -442,6 +477,25 @@ class FormFieldValue(OntologyElement):
     elementType: ElementTypeEnum = Field(ElementTypeEnum.form, frozen=True)
     allowed_tags: List[str] = Field(["input"], frozen=True)
 
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.text = self.additional_attributes.get("value", "")
+
+    def to_html(self, add_children=True) -> str:
```

Comment:
Hmm, because of current behavior of `to_html`, we're rendering text as content, so it is needed here - it is making sure that even if we have `self.text`, it will not be returned in HTML